tests/requirements-testing.txt

```diff
@@ -9,3 +9,4 @@ pytest-sugar==0.9.4
 pytest-timeout==2.1.0
 twine==3.7.1
 pytz==2021.3
+freezegun==1.2.1
```

Comment:
I'd rather not add another dependency for one test, if at all possible, better to just use `now() + x`.